### PR TITLE
Add README for unet layers.

### PR DESCRIPTION
**Pull request by yichunk**
Add README for unet layers.

BUG=None

### CODE REVIEW

File: ai_edge_torch/generative/layers/unet/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# ODML UNet Layers
+Common Pytorch building blocks to re-author UNet based models.
```
Comment (haozha111) [inline ai_edge_torch/generative/layers/unet/README.md:2]: PyTorch

File: ai_edge_torch/generative/layers/unet/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Blocks 2D layers
+`blocks_2d.py` provide common building blocks used in AutoEncoder and general UNet-like models. Each block has a corresponding config class provided in `model_config.py`, and the block layer is initialized with the config class. `blocks_2d.py` provide the following blocks:
```
Comment (haozha111) [inline ai_edge_torch/generative/layers/unet/README.md:5]: provides

Comment (yichunk) [reply ai_edge_torch/generative/layers/unet/README.md]: Done

File: ai_edge_torch/generative/layers/unet/README.md (hunk 1 above)
Comment (yichunk) [reply ai_edge_torch/generative/layers/unet/README.md]: Done

**Review by haozha111 — APPROVED** [review]

**Pull request merged**